src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -223,14 +231,61 @@ internal MetricPointStatus MetricPointStatus
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public readonly HistogramBuckets GetHistogramBuckets()
         {
-            if (this.aggType != AggregationType.Histogram && this.aggType != AggregationType.HistogramSumCount)
+            if (this.aggType != AggregationType.Histogram &&
+                this.aggType != AggregationType.HistogramSumCount &&
+                this.aggType != AggregationType.HistogramMinMax &&
+                this.aggType != AggregationType.HistogramSumCountMinMax)
             {
                 this.ThrowNotSupportedMetricTypeException(nameof(this.GetHistogramBuckets));
             }
 
             return this.histogramBuckets;
         }
 
+        /// <summary>
+        /// Gets the minimum value of the histogram associated with the metric
+        /// point if present.
+        /// </summary>
+        /// <param name="min">Output variable for the minimum value.</param>
+        /// <remarks>
+        /// Applies to <see cref="MetricType.Histogram"/> metric type.
+        /// </remarks>
+        /// <returns>A minimum value exists.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public bool TryGetHistogramMin(out double min)
+        {
+            if (this.aggType != AggregationType.HistogramMinMax &&
```

Comment:
we should not throw for any of the histogram type

Reply:
Okay, I shall return false instead